tests/bricks/test_conv.py

```diff
@@ -51,3 +54,65 @@ def test_max_pooling():
     pool.input_dim = (x_size, y_size)
     pool.get_dim('output') == (num_channels, x_size / pool_size + 1,
                                y_size / pool_size + 1)
+
+
+class ConvNN(Sequence, Initializable, Feedforward):
+    """Several convolutional layers
+
+    """
+    def __init__(self, conv_activations, input_dim, filter_sizes,
+                 feature_maps, pooling_sizes, conv_step=None, **kwargs):
+        if conv_step == None:
+            self.conv_step = (1, 1)
```

Comment:
Tuples are immutable, so it's okay to do `conv_step=(1, 1)`, which will make the function signature a bit clearer in the documentation.